src/locale/bn/_lib/formatLong/index.js

```diff
@@ -15,8 +15,8 @@ var timeFormats = {
 }
 
 var dateTimeFormats = {
-  full: "{{date}} 'বেলা' {{time}}",
-  long: "{{date}} 'বেলা' {{time}}",
+  full: "{{date}} {{time}} 'সময়'",
```

Comment:
This format will result `18/05/2018 7:32 পূর্বাহ্ন সময়`. Do you think we should move `সময়` before the time?

Reply:
I followed momentjs in this case.
<img width="348" alt="screen shot 2018-08-18 at 9 12 23 pm" src="https://user-images.githubusercontent.com/9896958/44298724-75cb9e80-a32b-11e8-857a-31de24fec698.png">